Zend/zend.h

```diff
@@ -77,7 +77,7 @@
 #define ZEND_TSRMLS_CACHE
 #endif
 
-ZEND_TSRMLS_CACHE_EXTERN();
+ZEND_TSRMLS_CACHE_EXTERN()
```

Comment:
Okay; not sure I'd feel comfortable changing that just yet, so I've reverted this one.